projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -35,46 +36,27 @@ const GLOBAL_MSG_QUOTE_REQUEST_NOT_POSSIBLE =
 /**
  * Selectors
  */
-const itemCounterSelector = 'cx-item-counter';
-const inputSelector = ' input';
-const quoteListSelector = 'cx-quote-list';
-const codeCellSelector = ' td.cx-code';
-const statusCellSelector = 'td.cx-status';
-const rowSelector = ' tr';
-const actionsLinkSelector = 'cx-quote-actions-link';
-const headerOverviewSelector = 'cx-quote-header-overview';
-const cardBodySelector = headerOverviewSelector + ' .cx-container .card-body';
-const cardParagraphSelector = cardBodySelector + ' .cx-card-paragraph-title';
-const commentsSelector = 'cx-quote-comments';
-const commentsMsgSelector = commentsSelector + ' .cx-message-input';
-const messagingSelector = 'cx-messaging';
-const messagingCardSelector = ' .cx-message-card';
-const messagingCardChildSelector =
-  commentsSelector + messagingCardSelector + ':nth-child';
-const itemsSelector = 'cx-quote-items';
-const itemListRowSelector = itemsSelector + ` .cx-item-list-row:nth-child`;
-const actionsByRoleSelector = 'cx-quote-actions-by-role';
-const btnSelector = 'button';
-const primaryBtnSelector = ' button.btn-primary';
-const secondaryBtnSelector = ' button.btn-secondary';
-const primaryBtnActionsByRoleSelector =
-  actionsByRoleSelector + primaryBtnSelector;
-const secondaryBtnActionsByRoleSelector =
-  actionsByRoleSelector + secondaryBtnSelector;
-const requestQuoteBtnSelector = 'cx-quote-request-button button';
-const accountPageTemplateSelector = '.AccountPageTemplate';
-const confirmDialogSelector = 'cx-quote-actions-confirm-dialog';
-const globalMsgSelector = 'cx-global-message';
-const sellerEditSelector = 'cx-quote-header-seller-edit';
+const addToCartComponentSelector = 'cx-add-to-cart';
+const quoteListComponentSelector = 'cx-quote-list';
+const actionsLinkComponentSelector = 'cx-quote-actions-link';
+const headerOverviewComponentSelector = 'cx-quote-header-overview';
+const commentsComponentSelector = 'cx-quote-comments';
+const quoteItemsComponentSelector = 'cx-quote-items';
+const quoteActionsByRoleComponentSelector = 'cx-quote-actions-by-role';
+const quoteRequestButtonComponentSelector = 'cx-quote-request-button';
+const itemCounterComponentSelector = 'cx-item-counter';
+const actionsConfirmDialogComponentSelector = 'cx-quote-actions-confirm-dialog';
+const messagingComponentSelector = 'cx-messaging';
+const quoteHeaderSellerEditComponentSelector = 'cx-quote-header-seller-edit';
 
 /**
  * Sets quantity.
  */
 export function setQuantity(quantity: string): void {
```

Comment:
```suggestion
export function setAddToCartQuantity(quantity: string): void {
```
maybe we should rename this so its clear that this relates to the addToCart popup?